ultravox/tools/wrappers.py

```diff
@@ -0,0 +1,60 @@
+import hashlib
```

Comment:
might want to call this `caching.py`. Also, don't need to do this now, but we're probably going to want to push ds_tool, tts, caching, templates down into their own ds_tool folder.

Reply:
Yeah makes sense. Will do.

Reply:
Done.